app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue

```diff
@@ -12,92 +12,101 @@
     @click="cardClick(chat)"
     @contextmenu="openContextMenu($event)"
   >
-    <label v-if="hovered || selected" class="checkbox-wrapper" @click.stop>
-      <input
-        :value="selected"
-        :checked="selected"
-        class="checkbox"
-        type="checkbox"
-        @change="onSelectConversation($event.target.checked)"
-      />
-    </label>
-    <thumbnail
-      v-if="bulkActionCheck"
-      :src="currentContact.thumbnail"
-      :badge="inboxBadge"
-      class="columns"
-      :username="currentContact.name"
-      :status="currentContact.availability_status"
-      size="40px"
-    />
-    <div class="conversation--details columns">
-      <div class="conversation--metadata">
-        <inbox-name v-if="showInboxName" :inbox="inbox" />
-        <div class="conversation-metadata-attributes">
-          <span
-            v-if="showAssignee && assignee.name"
-            class="label assignee-label text-truncate"
+    <div class="conversation--message-context  w-full">
+      <div class="conversation--first-row  w-full">
+        <div class="conversation--contact-meta">
+          <label
+            v-if="hovered || selected"
+            class="checkbox-wrapper"
+            @click.stop
           >
-            <fluent-icon icon="person" size="12" />
-            {{ assignee.name }}
+            <input
+              :value="selected"
+              :checked="selected"
+              class="checkbox"
+              type="checkbox"
+              @change="onSelectConversation($event.target.checked)"
+            />
+          </label>
```

Comment:
If [this](https://github.com/chatwoot/chatwoot/pull/6988/files#r1183405586) change is intentional, we could remove this or hide this permanently

Reply:
@scmmishra Not intentional, I have reverted the changes.